CHANGELOG.md

```diff
@@ -13,6 +13,18 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [0.3.0] - 24-04-23
+
+### Fixed
+
+* In the CLI, `dm topython` the argument `name` is now a option instead of a positional argument. This matched the
+  documentation in the README.
+
+### Changed
+
+* Renamed `DomainModelAPI.create` to `DomainModelAPI.apply`, to reflect the usage of the underlying endpoint.
```

Comment:
@doctrino Out of curiosity, where is `apply` found in underlying endpoints? Asking because I never saw it and it seems like In should have 😕 
https://api.cognitedata.com/api/v1/projects/{project}/models/instances